packages/cozy-stack-client/src/JobCollection.js

```diff
@@ -41,15 +41,17 @@ class JobCollection {
    * @param  {string} workerType - Ex: "konnector"
    * @param  {object} args - Ex: {"slug": "my-konnector", "trigger": "trigger-id"}
    * @param  {object} options - creation options
+   * @param  {boolean} manual -  Manual execution
```

Comment:
```suggestion
   * @param  {boolean} [manual=false] -  Manual execution
```
According to https://jsdoc.app/tags-param.html

Reply:
actually, I think we should uniformize. Either use default param  like you did (I prefer that), or use like line 53, the `or` operator.